bitnami/influxdb/values.yaml

```diff
@@ -0,0 +1,475 @@
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets (secrets must be manually created in the namespace)
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+  ## InfluxDB Relay Configuration
+  ## Specify content for relay.toml
+  ## Alternatively, you can put your relay.toml under the files/conf/ directory
+  ##
+  configuration: |-
+    [[http]]
+    # Name of the HTTP server, used for display purposes only.
+    name = "relay-server"
+
+    # TCP address to bind to, for HTTP server.
+    bind-addr = "0.0.0.0:9096"
+
+    # Array of InfluxDB instances to use as backends for Relay.
+    output = [
+        {{- $influxdbReplicaCount := int .Values.influxdb.replicaCount }}
+        {{- $influxdbFullname := include "influxdb.fullname" . }}
+        {{- $influxdbHeadlessServiceName := printf "%s-headless" (include "influxdb.fullname" .) }}
+        {{- $releaseName := .Release.Namespace }}
+        {{- range $e, $i := until $influxdbReplicaCount }}
+        { name="{{ $influxdbFullname }}-{{ $i }}", location="http://{{ $influxdbFullname }}-{{ $i }}.{{ $influxdbHeadlessServiceName }}.{{ $releaseName }}.svc.cluster.local:8086/write", timeout="10s"},
+        {{- end }}
+    ]
+
+  ## ConfigMap with InfluxDB Relay configuration
+  ## NOTE: This will override relay.configuration
+  ##
+  # existingConfiguration:
+
+  ## Number of InfluxDB Relay replicas to deploy
+  ##
+  replicaCount: 1
+
+  ## Update strategy, can be set to RollingUpdate or OnDelete by default.
+  ## ref: https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets
+  ##
+  updateStrategy: RollingUpdate
+
+  ## Pod AntiAffinity
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  antiAffinity: soft
+
+  ## Node Affinity. The value is evaluated as a template.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature
+  ##
+  nodeAffinity: {}
+
+  ## Node labels for pod assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for pod assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: {}
+
+  ## K8s Security Context for InfluxDB pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    fsGroup: 1001
+    runAsUser: 1001
+
+  ## InfluxDB Relay pods' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources: {}
```

Comment:
Change resources so it looks like this

```
  resources:
    # We usually recommend not to specify default resources and to leave this as a conscious
    # choice for the user. This also increases chances charts run on environments with little
    # resources, such as Minikube. If you do want to specify resources, uncomment the following
    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
    limits: {}
    #   cpu: 100m
    #   memory: 128Mi
    requests: {}
    #   cpu: 100m
    #   memory: 128Mi
```

This avoids this linter issue https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments_indentation